README

```diff
@@ -46,7 +46,9 @@ to use it. Run
   make
 
 to make installation files. Then copy gather script somewhere you want
-to have it, preferably in some directory from PATH. gather reads its
+to have it, preferably in some directory from PATH. Or you can look into
+a chef cookbook https://github.com/AlexJakeGreen/gatherit-cookbook for
+automatical installation of the script. gather reads its
```

Comment:
Could you pleas add a section "2.1 Installing with Chef" (or something like this) in the end of "2.Installation and configuration" section and add this info there?